src/Aspire.Hosting.Azure/Aspire.Hosting.Azure.csproj

```diff
@@ -14,6 +14,7 @@
     <Compile Include="..\Aspire.Hosting\Dcp\Process\ProcessSpec.cs" Link="Provisioning\Utils\ProcessSpec.cs" />
     <Compile Include="..\Aspire.Hosting\Dcp\Process\ProcessUtil.cs" Link="Provisioning\Utils\ProcessUtil.cs" />
     <Compile Include="..\Shared\Cosmos\CosmosConstants.cs" Link="Shared\CosmosConstants.cs" />
+    <Compile Include="$(SharedDir)ResourceGroupNameHelpers.cs" Link="shared\ResourceGroupNameHelpers.cs" />
```

Comment:
Does this need to be in a `Shared` directory? It is only used by 1 project.

Reply:
Moved to Aspire.Hosting.Azure